spec.html

```diff
@@ -46826,16 +46832,19 @@ <h1>
             1. Assert: If we return here, the generator either threw an exception or performed either an implicit or explicit return.
             1. Remove _acGenContext_ from the execution context stack and restore the execution context that is at the top of the execution context stack as the running execution context.
             1. Set _acGenerator_.[[GeneratorState]] to ~completed~.
-            1. NOTE: Once a generator enters the ~completed~ state it never leaves it and its associated execution context is never resumed. Any execution state associated with _acGenerator_ can be discarded at this point.
+            1. NOTE: Once a generator enters the ~completed~ state it never leaves it and there is never again a transfer of control to its associated execution context. Any execution state associated with _acGenerator_ can be discarded at this point.
             1. If _result_.[[Type]] is ~normal~, then
-              1. Let _resultValue_ be *undefined*.
+              1. Let _finalCompletion_ be NormalCompletion(CreateIterResultObject(*undefined*, *true*)).
             1. Else if _result_.[[Type]] is ~return~, then
-              1. Let _resultValue_ be _result_.[[Value]].
+              1. Let _finalCompletion_ be NormalCompletion(CreateIterResultObject(_result_.[[Value]], *true*)).
             1. Else,
               1. Assert: _result_.[[Type]] is ~throw~.
-              1. Return ? _result_.
-            1. Return CreateIterResultObject(_resultValue_, *true*).
-          1. Set the code evaluation state of _genContext_ such that when evaluation is resumed for that execution context, _closure_ will be called with no arguments.
+              1. Let _finalCompletion_ be _result_.
+            1. Let _callerContext_ be the running execution context.
+            1. Transfer control from _acGenContext_ to _callerContext_, passing _finalCompletion_.
+            1. NOTE: The above step never completes, because control is never transferred back to _acGenContext_.
```

Comment:
```suggestion
            1. NOTE: This step is never reached because control is never transferred back to _acGenContext_.
```

I don't want to have to think about what it means for a step to complete.

Reply:
Done.